devenv/local/docker-compose/docker-compose.yml

```diff
@@ -56,8 +56,9 @@ services:
     ports:
       - 5432:5432
     environment:
-      - POSTGRES_USER=postgres
-      - POSTGRES_PASSWORD=postgres
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: signer
```

Comment:
Oh, the postgres database is always created so adding the `POSTGRES_DB` environment variable here creates an additional database. For some reason I thought that the postgres database wasn't always created in the postgres docker image.

Reply:
This was just an observation, no action required.

Reply:
IIrc it actually changes the name of the default DB from `postgres` to the name given here, which in general shouldn't be an issue here for testing I think?